src/PrestaShopBundle/Resources/views/Admin/Sell/CustomerService/CustomerThread/Block/list_header_html.twig

```diff
@@ -0,0 +1,74 @@
+{# **
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * #}
+
+{# {% extends file="helpers/list/list_header.tpl" %} #}
+
+{% block leadin %}
```

Comment:
```suggestion
{% block leading %}
```

Reply:
not sure about this part. I can see that other page headers are extending 
{extends file="helpers/list/list_header.tpl"}
and it has block leadin 
{if !$simple_header}
	<div class="leadin">
		{block name="leadin"}{/block}
	</div>
{/if}

I don't really understand much about templates so not sure if i even need to extend this part but i m trying to keep the same logic as other parts.